.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "avocado-static-checks"]
 	path = avocado-static-checks
-	url = https://github.com/avocado-framework/avocado-static-checks
+	url = https://github.com/PaulYuuu/avocado-static-checks
+	branch = pre-commit
```

Comment:
This is a revision, taking my personal `avocado-static-checks` branch as an example, we can store `.pre-commit-config.yaml` and/or `commitlint.config.mjs` in `avocado-static-checks`, then link them in the current root, and pre-commit GitHub Action can use it without error by adding 
```
with:
    submodules: recursive
```